examples/ai-image-generator/src/assets/style.css

```diff
@@ -0,0 +1,69 @@
+@import "mirotone/dist/styles.css";
+
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
+
+.grid-wrapper {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  gap: 10px; /* Optional: Add gap between elements */
+}
+
+body {
+  display: flex;
+}
+
+#root {
+  overflow: auto;
+  padding: var(--space-medium);
+}
+
+.button {
+  width: 300px;
```

Comment:
100%?

Reply:
Please let me know if this is a blocker - then I can maybe take more time to implement this with cs1 ce12 syntax. if it's not a blocker will make video / merge as is then we can make a PR to update this in the future

Reply:
Fine, keep as is! styling isnt the point of this demo